cli/replicaset/configsource.go

```diff
@@ -10,15 +10,25 @@ import (
 	libcluster "github.com/tarantool/tt/lib/cluster"
 )
 
-// KeyPicker picks a key to patch.
-type KeyPicker func([]string, bool, string) (int, error)
-
-// path describes a path of target with its depth level in config.
-type path struct {
-	path  []string
-	depth int
+// pickPatchKeyOpts describes options for selecting a key to patch
+// the config.
+type PickPatchKeyOpts struct {
+	// pTargets describes a cluster config patch target that
+	// will be converted to Keys.
+	pTargets []patchTarget
+	// Keys is a destination for patching a key in storage.
+	Keys []string
+	// Force is an option for picking the first passed key
+	// if it is true.
+	Force bool
+	// PathMsg is a path destination in config to patch. If it is not
+	// empty path displays for user.
+	PathMsg string
 }
```

Comment:
To be honest, this is turning into something scary. It would be better to think about another design here.

Reply:
Deleted opts struct and passed args with enumeration how it was.